ts/annotations/annotations.src.ts

```diff
@@ -0,0 +1,1608 @@
+            addAnnotation(userOptions: AnnotationsOptions, redraw?: boolean): void;
+            drawAnnotations(): void;
+            initAnnotation(userOptions: AnnotationsOptions): Annotation;
+            removeAnnotation(idOrAnnotation: (number|string|Annotation)): void;
+        }
+        interface AnnotationControllableLabel {
+            itemType: 'label';
+        }
+        interface AnnotationControlPointEventsOptionsObject {
+            drag?: Function;
+        }
+        interface AnnotationControlPointOptionsObject {
+            draggable?: undefined;
+            events: AnnotationControlPointEventsOptionsObject;
+            height: number;
+            index?: number;
+            positioner: AnnotationControlPointPositioner;
+            style: CSSObject;
+            symbol: string;
+            visible: boolean;
+            width: number;
+        }
+        interface AnnotationControlPointPositioner {
+            (target: AnnotationControllable): PositionObject;
+        }
+        type AnnotationDraggableValue = (''|'x'|'y'|'xy');
+        type AnnotationLabelType = AnnotationControllableLabel;
+        type AnnotationShapeType = AnnotationControllable;
+        interface AnnotationMockPointOptionsObject {
+            x: number;
+            y: number;
+            xAxis?: (number|string|Axis|null);
+            yAxis?: (number|string|Axis|null);
+        }
+        interface AnnotationPoint extends Point {
+            series: AnnotationSeries;
+        }
+        interface AnnotationSeries extends Series {
+            chart: AnnotationChart;
+            points: Array<AnnotationPoint>;
+        }
+        interface AnnotationsEventsOptions {
+            afterUpdate?: EventCallbackFunction<Annotation>;
+            add?: EventCallbackFunction<Annotation>;
+            remove?: EventCallbackFunction<Annotation>;
+        }
+        interface AnnotationsLabelOptions extends AnnotationControllableOptionsObject {
+            align: AlignValue;
+            allowOverlap: boolean;
+            backgroundColor: ColorType;
+            borderColor: ColorType;
+            borderRadius: number;
+            borderWidth: number;
+            className: string;
+            crop: boolean;
+            distance?: number;
+            format?: string;
+            formatter: FormatterCallbackFunction<Point>;
+            overflow: DataLabelsOverflowValue;
+            padding: number;
+            shadow: (boolean|ShadowOptionsObject);
+            shape: SymbolKeyValue;
+            style: CSSObject;
+            text?: string;
+            useHTML: boolean;
+            verticalAlign: VerticalAlignValue;
+            x: number;
+            y: number;
+        }
+        interface AnnotationsLabelsOptions extends AnnotationsLabelOptions {
+            point?: (string|AnnotationMockPointOptionsObject);
+        }
+        interface AnnotationsOptions extends AnnotationControllableOptionsObject {
```

Comment:
I'm not sure about this one. Are really `AnnotationsOptions` extended by `AnnotationControllableOptionsObject`? I trust you here, so could you double check this?

Reply:
Well yesterday it was the conclusion, as all controllables take AnnotationsOptions, but I really do not know. It is just the typing that leads to this conclusion.

Reply:
Ah, right, this `x`/`y` options that actually works. Thanks!